src/node_file.cc

```diff
@@ -973,8 +1034,8 @@ void Access(const FunctionCallbackInfo<Value>& args) {
   const int argc = args.Length();
   CHECK_GE(argc, 2);
 
-  CHECK(args[1]->IsInt32());
-  int mode = args[1].As<Int32>()->Value();
+  Maybe<int> mode = GetValidMode(env, args[1], UV_FS_ACCESS);
+  if (mode.IsNothing()) return;
```

Comment:
Not wrong but you can also write this as:
```c++
Maybe<int> maybe_mode = GetValidMode(env, args[1], UV_FS_ACCESS);
int mode;
if (!maybe_mode.To(&mode)) return;
```
And then you don't have to call .FromJust() repeatedly. Eliminates the "have something?" check that method does.

Reply:
This is better, I will change. Thanks.